### ISSUE TITLE
TileLayer not shown when detectRetina is true

### ISSUE THREAD

**Issue by petrot** [Leaflet/Leaflet#6007]
## How to reproduce

- Leaflet version I'm using: 1.3
- Browser (with version) I'm using: Chrome 63.0.3239.132
- OS/Platform (with version) I'm using: OSX HighSierra 10.13.2 (17C205)

## What behaviour I'm expecting and which behaviour I'm seeing

With 'detectRetina: true', the map can't load the tiles. Expected behaviour: load tiles :)

## Minimal example reproducing the issue

`<script src="https://unpkg.com/leaflet@1.3.0/dist/leaflet.js"></script>`
```
var map = new L.Map('leaflet', {
	layers: [ 
		new L.TileLayer('//{s}.tile.openstreetmap.org/{z}/{x}/{y}.png', {
		  'detectRetina': true,
		})
	],
	center: [0, 0],
	zoom: 0
});
```

**Comment by cherniavskii** [comment]
Fixed by #6006 

It's kind of critical regression, I'll try to publish 1.3.1 as soon as possible

**Comment by petrot** [comment]
Thank you!

**Comment by cherniavskii** [comment]
Fix available in Leaflet 1.3.1 http://leafletjs.com/2018/01/18/leaflet-1.3.1.html

**Comment by petrot** [comment]
Thank you!! 👍  :)
